components/ui/AppDataTable.vue

```diff
@@ -53,6 +53,10 @@ export default class AppDataTable extends Vue {
 .bx--data-table th {
   color: $black-100;
   background-color: $cool-gray-20;
+
+  &[aria-sort] {
```

Comment:
As you can read [here](https://www.w3.org/TR/wai-aria-practices-1.1/examples/grid/dataGrids.html), `aria-sort` is an attribute that gives sortable capabilities information about the column. That is why I think this is not the best place to add the padding. Instead of this, you can add the `padding` line inside the `.bx--data-table th` definition

Reply:
@y4izus - I tried adding the padding there first, but it was getting overwritten Carbon's style... 
![Screen Shot 2021-04-07 at 7 44 00 AM (2)](https://user-images.githubusercontent.com/6276074/113877013-87307600-9775-11eb-8eab-8ab331d3f30e.png)


and I didn't want to use `!important`.

Thoughts?

Reply:
@y4izus - issue created here:

https://github.com/carbon-design-system/carbon-components-vue/issues/1161

Reply:
thank you very much!